projects/shared/src/lib/view/brief/person-brief/person-brief.component.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2019 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Component, Input } from '@angular/core';
+import { ResultItem } from '@rero/ng-core';
+import { ContributionTypePipe } from '../../../pipe/contribution-type.pipe';
+
+@Component({
+  selector: 'shared-person-brief',
+  templateUrl: './person-brief.component.html'
+})
+export class PersonBriefComponent implements ResultItem {
+
+  @Input() record: any;
```

Comment:
COSMETIC : missing docstring.... not really important for me

Reply:
done